docusaurus.config.js

```diff
@@ -16,7 +16,8 @@ module.exports = {
   plugins: [
     // ["plugin-image-zoom"],
     require.resolve('@cmfcmf/docusaurus-search-local'),
-    '@docusaurus-terminology/parser'
+    '@docusaurus-terminology/parser',
+    'docusaurus-plugin-sass'
```

Comment:
The changes here are for adding sass preprocessor to docusaurus, excluding the api docs source from docusaurus processing (we only want to process the generated docs) and changing the custom.css to support sass.